src/core/components/model-example.jsx

```diff
@@ -28,6 +28,10 @@ export default class ModelExample extends React.Component {
       activeTab = "example"
     }
 
+    if (!schema) {
```

Comment:
Let me know if we should merge this with one of the other `if` checks, as there already were two that only set `activeTab = "example"`

Reply:
Relating to https://github.com/swagger-api/swagger-ui/issues/9666#issuecomment-1980890003, perhaps we could do:

`let activeTab = defaultModelRendering === "model" && schema ? "model" : "example"`

I think this would also get rid of 
```
  if (defaultModelRendering !== "example" && defaultModelRendering !== "model") {
      activeTab = "example"
  }
```

Reply:
Yes, we can make this better, but let's do it in separate PR.